### PR TITLE
guvnor-rest-backend: typo fix

### PR DESCRIPTION
**Pull request by mrietveld**
`projectReq.setName(project.getProjectName());` is already called 2 lines below `projectReq.setName(projectGAV.getArtifactId());`..

### CODE REVIEW

**Comment by mrietveld** [comment]
See [here](https://github.com/droolsjbpm/guvnor/pull/216/files?w=1#diff-6fd862eeccdda5a8929d51951ec6b6e4L289).

File: guvnor-rest/guvnor-rest-backend/src/main/java/org/guvnor/rest/backend/ProjectResource.java [hunk 1]
```diff
@@ -275,27 +275,26 @@ public Response createProject(
...
         Set<Project> projects = projectService.getProjects(repository, repository.getCurrentBranch());
-        
+
         List<ProjectResponse> projectRequests = new ArrayList<ProjectResponse>(projects.size());
-        for( Project project : projects ) { 
+        for( Project project : projects ) {
            ProjectResponse projectReq = new ProjectResponse();
            GAV projectGAV = project.getPom().getGav();
            projectReq.setGroupId(projectGAV.getGroupId());
-           projectReq.setName(projectGAV.getArtifactId());
```
Comment (psiroky) [inline guvnor-rest/guvnor-rest-backend/src/main/java/org/guvnor/rest/backend/ProjectResource.java]: Shouldn't this be `projectReq.setArtifactId(projectGAV.getArtifactId())`?

Comment (mrietveld) [inline guvnor-rest/guvnor-rest-backend/src/main/java/org/guvnor/rest/backend/ProjectResource.java]: @psiroky. No. Please always first take a look at the actual code before making comments... :/ 

See the following classes: 

- [ProjectResponse](https://github.com/droolsjbpm/guvnor/blob/master/guvnor-rest/guvnor-rest-client/src/main/java/org/guvnor/rest/client/ProjectResponse.java)
- [ProjectRequest](https://github.com/droolsjbpm/guvnor/blob/master/guvnor-rest/guvnor-rest-client/src/main/java/org/guvnor/rest/client/ProjectRequest.java)
- [Entity](https://github.com/droolsjbpm/guvnor/blob/master/guvnor-rest/guvnor-rest-client/src/main/java/org/guvnor/rest/client/Entity.java)

Comment (psiroky) [inline guvnor-rest/guvnor-rest-backend/src/main/java/org/guvnor/rest/backend/ProjectResource.java]: Fair enough. Sorry for the noise.

**Pull request closed without merging**